src/olympia/abuse/cinder.py

```diff
@@ -343,7 +343,7 @@ def get_attributes(self):
         # promoted in any way, but we don't care about the promotion being
         # approved for the current version, it would make more queries and it's
         # not useful for moderation purposes anyway.
-        promoted_group = self.addon.promoted_groups(currently_approved=False)
+        promoted_groups = self.addon.promoted_groups(currently_approved=False)
```

Comment:
changing the variable name across all of the files from `promoted_group` to `promoted_groups` makes the diff harder to read. That would be a good candidate for a separate commit perhaps the first commit.